lib/core/ui/speaker_detail_content.dart

```diff
@@ -136,21 +131,18 @@ class _SpeakerInfo extends StatelessWidget {
                 ? MainAxisAlignment.center
                 : MainAxisAlignment.start,
             children: [
-              if (speaker.github != null)
```

Comment:
Bu kısımda konuşmacıların tüm sosyal medya linkleri bulunmadığı için kontrol eklediğimizi düşünüyorum. Tekrardan kontrol edebiliriz

Reply:
null gelme olasılığı burada olmadığı için gereksiz kullanım uyarısı aldığım için sildim.